### DISCUSSION TITLE
Option to disable endless scrolling

### DISCUSSION THREAD

**Discussion by yaomtc** [Docile-Alligator/Infinity-For-Reddit#913] [Ideas]
Right now, NoSurf For Reddit and RedReader are the only open source reddit clients with this feature. Reddit Is Fun also has this, but it's not open source.

**Comment by Docile-Alligator** [comment]
Do you mean you want to manually click a button to load the next page?

**Comment by yaomtc** [comment]
Yeah. That's how the mobile site works (which is now treated as "legacy" and it doesn't seem like it's ever getting fixes) and old.reddit. I don't like an endless, automatic flow of content, I prefer to just read a set portion, which a single page is perfect for.

**Comment by Docile-Alligator** [comment]
This will be difficult to add because the paging library will automatically handle pagination when you scroll down the posts. But I will check if there is an option to change this behavior though.

**Comment by jm355** [comment]
Could also just add an option to limit pages to X number of posts, and have it default to however many posts are on a page

**Comment by BeatLink** [comment]
Are there any updates on this feature?

**Comment by yaomtc** [comment]
This was turned into a discussion three days ago, according to the email I received (why it's not indicated here, I don't know; I assume you found this today and didn't know) and by not adding a comment at that time, the update is that there is no update, not until someone gets around to looking into solving it. I imagine this is not something that the Docile-Alligator needs and the number of people who are interested in such a feature is probably pretty small compared to some other feature ideas, of which there are four pages' worth.

**Reply by Wladefant** [reply]
Hey @yaomtc ,
right now we are restructuring the things around the project and are making feature requests more visible. Never give your hope up 😊. We will implement the things by priority (which is not yet clear) but also for example if a developer likes the feature 😄

**Reply by yaomtc** [reply]
@Wladefant, I meant for this to be a reply to @BeatLink's comment, I must have misclicked. Haven't given up on this idea. I just don't like to see emails for comments that are effectively just a "bump".

My issue with infinite scrolling is that I tend to lose track of time by focusing too intently on something inane. This is why I uninstalled TIkTok on my phone, why I use reddit's mobile site, and why I avoid using the "Home" feed on Twitter and stick to "Latest" and don't follow people who post a lot of tweets every day (I should probably just use twitter less). Pagination is a way I can put up a wall that makes me stop and think about what I'm doing. My ADHD is probably a factor here.

**Reply by Wladefant** [reply]
All right 😀, you are definitely correct with that 😏. 
And thank you for being such a long time user. Congrats
